pkg/components/components_test.go

```diff
@@ -0,0 +1,90 @@
+package components
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+const imageName = "the-image-name"
+
+var _ = Describe("Components Tests", func() {
+	Context("RelatedImage Test", func() {
```

Comment:
These three `It`s in this `Context` do the exact test but on different inputs.
please change the `Context` to `DescribeTable` with Entries. [example](https://github.com/kubevirt/cluster-network-addons-operator/blob/master/tools/bumper/bumper_test.go#L17)